test/parallel/test-stream-construct.js

```diff
@@ -0,0 +1,294 @@
+'use strict';
+
+const common = require('../common');
+const { Writable, Readable } = require('stream');
+const assert = require('assert');
+
+{
+  // Pass options as is to construct.
+
+  const options2 = {
+    test: 'foo',
+    construct: common.mustCall((options, callback) => {
+      assert.strictEqual(options, options2);
+      callback();
+    })
+  };
+  const w = new Writable(options2);
+  w.destroy();
+}
+
+{
+  // Pass options as is to construct.
+
+  const options2 = {
+    test: 'foo',
+    construct: common.mustCall((options, callback) => {
+      assert.strictEqual(options, options2);
+      callback();
+    }),
+    read() {}
+  };
+  const r = new Readable(options2);
+  r.destroy();
+}
+
+{
+  // Multiple callback
```

Comment:
Nit: I think we try to make comments capitalized and end with `.` recently. (same applies to the below test comments if you wish to change this)
```suggestion
  // Multiple callback.
```